src/main/java/com/openlattice/chronicle/data/ChronicleDataCollectionAppConfig.java

```diff
@@ -77,14 +79,14 @@ public UUID getAppsDictionaryEntitySetId() {
 
     @JsonProperty( SerializationConstants.ENTITY_SET_IDS )
     public Set<UUID> getAllEntitySetIds() {
-        return ImmutableSet.of(
+        return Sets.newHashSet(
```

Comment:
Arrays.asList(...) since you are going to filter and collect to set anyway (avoid unnecessary set construction).